index.js

```diff
@@ -153,6 +164,21 @@ module.exports.errorLogger = function (opts) {
                 }
             }
 
+            // obfuscate last in case we set something in our includesFn
+            if (obfuscate) {
+              for(var i in obfuscate) {
+                var key = obfuscate[i];
+                if (has(json, key)) {
+                  set(json, key, obfuscatePlaceholder);
+                }
+              }
+            }
+
+            // Set the short-body here in case we've modified the body in obfuscate
+            if (json && json.body) {
```

Comment:
With that change, you don't need the other short-body definition anymore.
Maybe you can set it to undefined up there: https://github.com/shutterstock/express-bunyan-logger/blob/ae774272409dccbf6c1b43a0a55d7c7b2d4d1f2f/index.js#L124